assignments/array-methods.js

```diff
@@ -57,27 +57,65 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
 let fullName = [];
 console.log(fullName);
+runners.forEach(function(runner){console.log(`Runner's first and last name is  ${ runner.first_name} ${runner.last_name} .`)})
+
+//---- Don't understand why it is not populating the array---
+//runners.forEach(function(runner){console.log(`Runner's first and last name is ${runner.first_name} ${runner.last_name}.`)}) 
+
+
+runners.forEach(index => fullName.push("Runner's first and last name is " + index.first_name + ' ' + index.last_name));
```

Comment:
On this, I'd recommend not using a full sentence. Just return the first name and the last name. According to the instructions, they need to be printed on a running bib. You wouldn't want to print a whole sentence on the bib, just the name! Otherwise, great job.